packages/android_alarm_manager/lib/android_alarm_manager.dart

```diff
@@ -70,10 +67,7 @@ class AndroidAlarmManager {
       return false;
     }
     final dynamic r = await _channel
-        // TODO(amirh): remove this on when the invokeMethod update makes it to stable Flutter.
-        // https://github.com/flutter/flutter/issues/26431
-        // ignore: strong_mode_implicit_dynamic_method
-        .invokeMethod('AlarmService.start', <dynamic>[handle.toRawHandle()]);
+        .invokeMethod<dynamic>('AlarmService.start', <dynamic>[handle.toRawHandle()]);
```

Comment:
```suggestion
    return _channel.invokeMethod<bool>('AlarmService.start', <dynamic>[handle.toRawHandle()]);
```

Reply:
Its not necessary to do the explicit cast, that is the point of the invokeMethod parameter